descriptions/css/properties/background-origin.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "background-origin": {
+        "__short_description": "The <strong><code>background-origin</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property sets the <em>background positioning area</em>. In other words, it sets the origin position of an image set with the <a href='https://developer.mozilla.org/docs/Web/CSS/background-image'><code>background-image</code></a> property."
+      }
```

Comment:
I think this is not very helpful. It describe the property in a bit of spec-speak "background positioning area" that it then immediately has to define. Also `image` is not strictly correct: it could 
also AFAIK be a color.

Could we say something like: "The CSS background-origin property sets the background's origin: from the border start, inside the border, or inside the padding."